client/core/core.go

```diff
@@ -230,6 +233,10 @@ func (dc *dexConnection) marketMap() map[string]*Market {
 		}
 		mktID := mkt.marketName()
 
+		for _, trade := range dc.marketQueuedTrades(mktID) {
+			mkt.Orders = append(mkt.Orders, trade.coreOrder())
```

Comment:
This is what I was referring to regarding a new field somewhere in the `User` struct (a new `Market.PendingOrders` field that is within the bigger `User` struct).  In https://github.com/decred/dcrdex/pull/1887#issuecomment-1279277001 you say "User is not stored on core", but I don't understand the meaning.

And if the queued orders are stored in a different type, we don't use `trade.coreOrder` and hopefully don't have to deal with the mutability issues mentioned in my other comment.  It can literally just be *copies* of an immutable `db.OrderMetaData` object. No need to modify `marketEpoch`, no other edge cases to check with the zero order time, etc.

And by putting pending orders in a completely separate field, there's no need for the new order status.

Reply:
> And by putting pending orders in a completely separate field, there's no need for the new order status.

Putting them a `Market.PendingOrders` field works. I was looking at adding support for viewing temp orders on the `order/oid` page but no longer think it is necessary.

Reply:
You could get away with just one Orders slice and no new order status.
I think they important changes are too avoid trackedTrades in the queuedOrders map